### ISSUE TITLE
Expected/actual should be swapped for ContainerBuilderHasParameterConstraint

### ISSUE THREAD

**Issue by WouterJ** [matthiasnoback/SymfonyDependencyInjectionTest#49]
```
...
There was 1 failure:

1) Symfony\Cmf\Bundle\RoutingBundle\Tests\Unit\DependencyInjection\CmfRoutingExtensionTest::testRouteBasepathsMerging with data set #2 (null, array('/cms/test'), array('/cms/routes', '/cms/test'), '/cms/routes')
The value of parameter "cmf_routing.dynamic.persistence.phpcr.route_basepaths" (Array &0 (
    0 => '/cms/routes'
    1 => '/cms/test'
)) does not match the expected value (Array &0 (
    0 => '/cms/test'
))
Failed asserting that Symfony\Component\DependencyInjection\ContainerBuilder Object &000000004ae804bd0000000034c3290d (
    ...
    'parameterBag' => Symfony\Component\DependencyInjection\ParameterBag\ParameterBag Object &000000004ae8069d0000000034
c3290d (
        'parameters' => Array &5 (
            ...
            'cmf_routing.dynamic.persistence.phpcr.route_basepaths' => Array &11 (
                0 => '/cms/test'
            )
            ...
    )
    ...
) has a parameter "cmf_routing.dynamic.persistence.phpcr.route_basepaths" with the given value.
```

As you can see, the actual value of the parameter is `array('/cms/test')`, while the expected was `array('/cms/routes', '/cms/test')`. The failure message says the opposite.

**Comment by xabbuh** [comment]
see #50

**Comment by matthiasnoback** [comment]
Also released, as v0.7.5.
